### PR TITLE
Stops the Weight Bench from being deleted when select wheel is closed

### LINKED ISSUE #12949: you can delete the workout benchs
### Description

basically you can thanos snap the workout benches from existance by simply right clicking on them twice or when the radial menu to change the weight is up according to Alog there is no runtimes so i have no idea why exactly this happens (after some banter mid making this i have found the sprite disappears not the table)

### Test Merges

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/sybil/server_side_modifications.dm

Round ID: 21088
BYOND Version: 515.1607
Server revision compiled on: 
Master commit: [56b2d7a380a518c970c11175aa526620825e50d5](https://github.com/tgstation/TerraGov-Marine-Corps//commit/56b2d7a380a518c970c11175aa526620825e50d5)
The following pull requests are currently test merged:
[#12908: 'Platform rework' by Lumipharon at commit e05eb53b77](https://github.com/tgstation/TerraGov-Marine-Corps//pull/12908)

Local commit: 449e6501896252299f2206274dd1a26fa8645af9
Server tools version: 5.12.0

### Reproduction Steps

right click the workout bench when you have the radial menu to change its weight up you can bring it back by clicking one of the radial menu options however if you close it the bench is forever gone.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by JackTheJackhammer**
## About The Pull Request

Makes it so that the Weight Bench no longer gets banished to the shadowrealm if you dont wish to change the weights.

## Why It's Good For The Game

The TGMC can now stop having to buy a new weight bench everytime someone changes their mind and banishes the Weight bench
Also fixes #12949

## Changelog
:cl:

fix: Weight Benches no longer disappear when unselecting the radial wheel.

/:cl:

### CODE REVIEW

File: code/game/objects/structures/benchpress.dm [hunk 1]
```diff
@@ -72,6 +72,8 @@
 	)
 	var/choice = show_radial_menu(user, src, radial_options, null, 64, null, TRUE, TRUE)
 	plates = text2num(choice)
+	if(!choice)
+		return
```
Comment (Lumipharon) [inline code/game/objects/structures/benchpress.dm:76]: shouldn't this be before plates is set?

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/benchpress.dm (hunk 1 above)
Comment (JackTheJackhammer) [reply code/game/objects/structures/benchpress.dm:76]: oh wait yeah you have a point

Comment (Lumipharon) [reply code/game/objects/structures/benchpress.dm:76]: @JackTheJackhammer it hasn't changed?

**Comment by Lumipharon** [comment]
it didn't change my man.

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**